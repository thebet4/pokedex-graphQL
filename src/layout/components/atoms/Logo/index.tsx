import { Image } from "react-native";

const Logo = () => {
  return <Image source={require("@core/assets/images/logo.png")} />;
};

export default Logo;
