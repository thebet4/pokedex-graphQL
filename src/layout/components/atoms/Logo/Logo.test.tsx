import * as React from "react";
import renderer from "react-test-renderer";

import Logo from "@layout/components/atoms/Logo";

it(`renders <Logo /> correctly`, () => {
  const tree = renderer.create(<Logo />).toJSON();

  expect(tree).toMatchSnapshot();
});
