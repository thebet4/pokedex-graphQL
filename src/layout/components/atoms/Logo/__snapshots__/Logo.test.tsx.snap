// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Logo /> correctly 1`] = `
<Image
  source={1}
/>
`;
